nav2_bringup/package.xml

```diff
@@ -19,6 +19,7 @@
   <exec_depend>navigation2</exec_depend>
   <exec_depend>nav2_common</exec_depend>
   <exec_depend>slam_toolbox</exec_depend>
+  <exec_depend>turtlebot3_gazebo</exec_depend>
```

Comment:
I think this is what we discussed that we intentionally did not select (?)

Reply:
The tutorials recommend installing `ros-humble-turtlebot3*`, which is bad due to the size limitations. As noted in the PR description, I checked the size of of just using `ros-humble-turtlebot3-gazebo`, and it's minuscule. Are you still asserting we should not add a dependency, even though it's small?

Reply:
Sorry, I thought something was different than it was, this is fine